packages/react/src/primitives/Tabs/TabsContainer.tsx

```diff
@@ -7,6 +7,9 @@ import { primitiveWithForwardRef } from '../utils/primitiveWithForwardRef';
 import { BaseTabsProps, TabsProps } from './types';
 import { View } from '../View';
 import { TabsContext } from './TabsContext';
+import { useStableId } from '../utils/useStableId';
+
+const whitespaceValue = '-'; // used to create valid HTML IDs
```

Comment:
Module scoped constants should be declared in SCREAMING_SNAKE_CASE

Reply:
Love the name SCREAMING_SNAKE_CASE